system/include/emscripten/val.h

```diff
@@ -620,7 +630,10 @@ class val {
     return v;
   }
 
-  EM_VAL handle;
+#if !defined(NDEBUG) && defined(_REENTRANT)
+  pthread_t thread;
+#endif
+  EM_VAL handle_;
```

Comment:
Why the underscore suffix?  If that is for private members why not also add that for `thread` above?

Reply:
TBH it was mostly for me to quickly notice error squiggles everywhere `handle` is used, but then decided to leave as-is as sort of a "private" notation, yeah. I'm happy to either rename back or rename `thread` too.

Reply:
Maybe lets just leave it without the underscore then?

Reply:
Renamed back.